docs/.vuepress/config.js

```diff
@@ -37,8 +37,8 @@ module.exports = {
               "components/search-bar.md",
               "components/button.md",
               "components/textfield.md",
+              "components/scrollview.md",
               "components/slider.md",
-              "components/textview.md",
```

Comment:
why did you delete textview ?

Reply:
Because of conflict.After resolve conflict there was 2 textfield component and i deleted